test/e2e/tests/tokens/nft/erc721-interaction.spec.js

```diff
@@ -38,19 +38,16 @@ describe('ERC721 NFTs testdapp interaction', function () {
 
         // Notification
         await driver.waitUntilXWindowHandles(3);
-        const windowHandles = await driver.getAllWindowHandles();
-        const [extension] = windowHandles;
```

Comment:
Having the extension window as a variable and keep referencing it steps later can cause the context to be invalidated. This is not recommended pattern and instead we should get the windows at the moment we want to switch to each of them